src/Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI/Extension/SetupHostExtensions.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Configurations.AppSettings.Extensions;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Abstractions;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Configurations;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Extensions;
+using Microsoft.Extensions.Configuration;
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI.Extension
```

Comment:
Can you change the directory name and namespace from `Extension` to `Extensions` by following the conventions?

Reply:
Changed to match request in new commit